XpringKit/ILP/IlpClient.swift

```diff
@@ -0,0 +1,44 @@
+/// A client that can get balances and send ILP payments on a Xpring connector.
+public class IlpClient {
+
+    private let decoratedClient: IlpClientDecorator
+
+    /// Initialize a new client with a configured URL
+    ///
+    /// - Parameters:
+    ///     - grpcUrl : The gRPC URL exposed by Hermes
+    public init(grpcUrl: String) {
+        decoratedClient = DefaultIlpClient(grpcUrl: grpcUrl)
+    }
+
+    /// Get the balance of the specified account on the connector.
+    ///
+    /// - Parameters:
+    ///     -  accountId The account ID to get the balance for.
+    ///     -  bearerToken Authentication bearer token.
+    /// - Returns: A Org_Interledger_Stream_Proto_GetBalanceResponse with balance information of the specified account
+    /// - Throws: An error If the given inputs were invalid, the account doesn't exist, or authentication failed.
+    public func getBalance(for accountId: String, bearerToken: String) throws -> Org_Interledger_Stream_Proto_GetBalanceResponse {
```

Comment:
expanded on this below but I'd change:
`bearerToken: String` -> `withAuthorization bearerToken: String`